spec/views/idv/doc_auth/upload.html.erb_spec.rb

```diff
@@ -11,20 +11,8 @@
     allow(view).to receive(:user_signing_up?).and_return(false)
   end
 
-  context 'with liveness enabled' do
-    it 'renders liveness content' do
-      allow(view).to receive(:liveness_checking_enabled?).and_return(true)
-      render template: 'idv/doc_auth/upload'
-
-      expect(rendered).to have_content(t('doc_auth.headings.upload_liveness_enabled'))
-      expect(rendered).to have_content(t('doc_auth.info.upload_liveness_enabled'))
-      expect(rendered).to have_content(t('doc_auth.headings.upload_from_phone_liveness_enabled'))
-    end
-  end
-
   context 'with liveness disabled' do
```

Comment:
We can probably pull this context, right? `with liveness disabled` doesn't exactly make sense in a world where liveness has been taken out

Reply:
Removed.